data/controls.xml

```diff
@@ -166,6 +166,8 @@
         <textureradioofffocus colordiffuse="color">path</textureradioofffocus>
         <textureradiooffnofocus colordiffuse="color">path</textureradiooffnofocus>
         <textureradiooffdisabled colordiffuse="color">path</textureradiooffdisabled>
+        <textureradioon colordiffuse="color">path</textureradioon>
+        <textureradiooff colordiffuse="color">path</textureradiooff>
```

Comment:
Sorry, I do not, the skin code I am cleaning up doesn't use it and I've never seen it used elsewhere so not sure if any of the radio on/off button textures can take a border attribute. I assumed it would be the same as focus and unfocus since it is supposed to be a shortcut for both on/off using the same image, at least according to the [wiki page](http://kodi.wiki/view/Radio_button_control).